scripts/nz/lds-nz.sh

```diff
@@ -1,24 +1,24 @@
 #!/bin/bash
 
-# 1. Download the zip file from https://data.linz.govt.nz/layer/779-nz-street-address-electoral/
+# 1. Download the zip file from https://data.linz.govt.nz/layer/3353-nz-street-address/
 #    into this directory
```

Comment:
this prior instruction `into this directory` isn't correct. Let's change it to match the working flow. I think it should say `into the /work folder`